scipy/interpolate/src/_fitpackmodule.c

```diff
@@ -15,6 +10,7 @@ static PyObject *fitpack_error;
 
 #define F_INT npy_int64
 #define F_INT_NPY NPY_INT64
+#define F_INT_MAX INTPTR_MAX
```

Comment:
For the case where `HAVE_ILP64` defined on 32-bit platform.

```suggestion
#define F_INT_MAX NPY_MAX_INT64
```